docs/discovery/dev/bos-loader.md

```diff
@@ -0,0 +1,62 @@
+---
+id: bos-loader
+title: BOS Loader
+---
+
+# BOS Loader
+
+In this article you'll learn how to develop, test, and deploy BOS components using CLI tools. You can use this workflow to tap into the colaboration, pull-request, and other GitHub benefits while still deploying components to the BOS. 
+
+[BOS Component Loader](https://github.com/near/bos-loader) serves a local directory of component files as a JSON payload properly formatted to be plugged into a BOS `redirectMap`. When paired with a viewer configured to call out to this loader, it enables local component development.
+
+## Development flow
+
+1. _(Optional)_ Download and install [bos-cli](https://github.com/FroVolod/bos-cli-rs/releases).
+
+   :::info
+   You need `bos-cli` if you have component code on the BOS already that you want to use or if you want to manage component deploys localy instead of the GitHub actions CI/CD.
```

Comment:
```suggestion
   You need `bos-cli` if you have component code on the BOS already that you want to use or if you want to manage component deploys locally instead of the GitHub actions CI/CD.
```